### PR TITLE
Remove useless code

### PR DESCRIPTION
**Pull request by yepninja**

### CODE REVIEW

File: lib/rules/function-name-case/index.js [hunk 1]
```diff
@@ -45,7 +45,6 @@ const rule = function(expectation, options, context) {
     }
 
     root.walkDecls(decl => {
-      let needFix = false;
```
Comment (vankop) [inline lib/rules/function-name-case/index.js]: there is some performance idea in this variable ( `parsed.toString()` is a bit expensive operation, see [stringify.js](https://github.com/TrySound/postcss-value-parser/blob/master/lib/stringify.js)), for instance when function name is correct, but `context.fix === true`, it is not necessary to perform `parsed.toString()`

**Comment by vankop** [comment]
@yepninja I have wrote my suggestion to improve performance

**Comment by jeddy3** [comment]
It seems like the `let needFix = false;` approach is a valid performance addition. Shall we close this PR?

**Comment by jeddy3** [comment]
Closing this PR. Feel free to reopen if there's more to discuss.

**Pull request closed without merging**